### PR TITLE
fix core dump & support DPDK 18.11 & update travis.yaml

### PR DESCRIPTION
**Pull request by yangye-huaizhou**
I merged the serveral commits and fixed the bugs I know so far. It works well now.

### CODE REVIEW

**Comment by groud** [comment]
Hi thanks for your contribution and sorry for my late response. Now than a new stable DPDK version is there, it is worth going to a newer version of DPDK. 

Is this PR ready to be merged ?

**Comment by yangye-huaizhou** [comment]
Yes, but it only supports DPDK version 18.11.
We still need a little more time to support the latest stable DPDK version.
And we also created a new branch, which is about the implementation of multiple ports and multiple queues.
Anyway, I will let you know when we are finished.

**Comment by groud** [comment]
OK sounds good, I will update the readme to explain this. 
Just a question before merging, is libnuma-dev a new dependency of DPDK ? why is it needed by the travis tests ?

**Comment by yangye-huaizhou** [comment]
Oh, you are right. It has already been depended by DPDK.
You can delete it.

**Comment by groud** [comment]
Could you amend the commit ? I can also do it myself, but it will take more time ^^

**Comment by groud** [comment]
Hmm, the CI doesn't pass anymore now :/ Maybe it way needed, but it looks like it's a DPDK header that is missing.

**Comment by yangye-huaizhou** [comment]
Sorry, I think it's not about the codes.
I try to clean the cache and rebuild the DPDK.

**Comment by groud** [comment]
Ah you were right, cleaning the cache made it work.
So could you: 
- Remove the libnuma dependancy again
- Squash all commits from the branch together to keep the git history clean

**Comment by yangye-huaizhou** [comment]
OK.
But I will try that a few hours later.

File: .travis.yml [hunk 1]
```diff
@@ -3,19 +3,19 @@ addons:
...
     packages:
     - libncurses-dev
-    - linux-headers-3.13.0-40-generic
+    - linux-headers-4.15.0-58-generic
 
 cache:
   directories:
     - $HOME/dpdk
-
+
```
Comment (groud) [inline .travis.yml]: You have a trailing whitespace here

**Comment by yangye-huaizhou** [comment]
/home/travis/dpdk/lib/librte_eal/linuxapp/eal/eal_memory.c:28:18: fatal error: numa.h: No such file or directory
4271compilation terminated.

I think we need to add libnuma-dev

**Comment by groud** [comment]
Ok, things look good to me now. Is everything ready on your side ?

**Comment by yangye-huaizhou** [comment]
Yes, I think it's ready to be merged.
And we need libnuma-dev to compile the newer version DPDK

**Comment by groud** [comment]
Ok thanks !

**Pull request merged**